internal/controller/standalone_pgadmin/statefulset.go

```diff
@@ -132,8 +134,10 @@ func statefulset(
 		}
 
 		collector.AddToPod(ctx, pgadmin.Spec.Instrumentation, pgadmin.Spec.ImagePullPolicy,
-			configmap, &sts.Spec.Template.Spec, volumeMounts, "", []string{}, false)
+			configmap, &sts.Spec.Template.Spec, volumeMounts, "", []string{LogDirectoryAbsolutePath}, false)
 	}
 
+	postgrescluster.AddTMPEmptyDir(&sts.Spec.Template)
```

Comment:
❓ Oh hmm. Does the STS still have a writable `/tmp` ?

Reply:
Yeah, I noticed that in most other cases, we don't add `/tmp` directly in our spec but through this func that adds it to all containers. 

I ran into this problem with pgadmin (where /tmp was only avail to the main container) when the collector tried to start up and write a PID file to /tmp. I thought about revising that to use a var rather than file (as we talked about when we merged that PR), but I also wanted the collector container to have access to /tmp anyway, just in case. (Could also do both.)